config/prrte_check_lsf.m4

```diff
@@ -121,6 +121,72 @@ AC_DEFUN([PRRTE_CHECK_LSF],[
+          #   lib64/libevent_pthreads.so: undefined reference to `event_mm_malloc_'
+          #   lib64/libevent_pthreads.so: undefined reference to `event_mm_free_'
+          #   lib64/libevent_pthreads.so: undefined reference to `evthread_set_id_callback'
+          #   lib64/libevent_pthreads.so: undefined reference to `evthread_set_lock_callbacks'
+          # Because it picked up -levent from LSF, but -levent_pthreads from Libevent.
+          #
+          # So look for a function that libevent_pthreads is looking for from libevent.so.
+          # If it does appears then we have the correct libevent.so, otherwise then we picked
+          # up the LSF version and a conflict has been detected.
+          # If the external libevent component used 'event_core' instead of 'event'
+          prrte_check_lsf_event_conflict=na
+          # Split libs into an array, see if -levent is in that list
+          prrte_check_lsf_libevent_present=`echo $LIBS | awk '{split([$]0, a, " "); {for (k in a) {if (a[[k]] == "-levent") {print a[[k]]}}}}' | wc -l`
+          AS_IF([test "$prrte_check_lsf_happy" = "no"],
+                [AS_IF([test "$prrte_check_lsf_libevent_present" != 0],
+                       [AS_IF([test "$prrte_check_lsf_libdir" = "" ],
+                              [],
+                              [LDFLAGS="$LDFLAGS -L$prrte_check_lsf_libdir"])
+                        # Note that we do not want to set LIBS here to include -llsf since
+                        # the check is not for an LSF library, but for the conflict with
+                        # LDFLAGS.
+                        AC_CHECK_LIB([event], [evthread_set_condition_callbacks],
+                                     [AC_MSG_CHECKING([for libevent conflict])
+                                      AC_MSG_RESULT([No. The correct libevent.so was linked.])
+                                      prrte_check_lsf_event_conflict=no],
+                                     [AC_MSG_CHECKING([for libevent conflict])
+                                      AC_MSG_RESULT([Yes. A wrong libevent.so was linked.])
+                                      prrte_check_lsf_event_conflict=yes])
+                       ],
+                       [AC_MSG_CHECKING([for libevent conflict])
+                        AC_MSG_RESULT([No. -levent is not being explicitly used.])
+                        prrte_check_lsf_event_conflict=na])],
+                [AC_MSG_CHECKING([for libevent conflict])
+                 AC_MSG_RESULT([No. LSF checks passed.])
+                 prrte_check_lsf_event_conflict=na])
+
+          AS_IF([test "$prrte_check_lsf_event_conflict" = "yes"],
+                [AC_MSG_WARN([===================================================================])
+                 AC_MSG_WARN([Conflicting libevent.so libraries detected on the system.])
+                 AC_MSG_WARN([])
+                 AC_MSG_WARN([A system-installed Libevent library was detected and the Open MPI])
```

Comment:
Urr...should be "PRRTE" and not "Open MPI"

Reply:
Ah yes. Thanks. I miss that when picking it over.